tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -56,6 +59,86 @@ public void CreateWwwAuthenticateResponse(string resource, string authorizationU
             Assert.IsNull(authParams.Error);
         }
 
+        [TestMethod]
+        [DataRow("AuthScheme", "" )]
+        [DataRow("AuthScheme", "realm = someRealm")]
+        [DataRow("AuthScheme", "token68")]

```

Comment:
I ran this test and I am not sure this is the correct result? The `RawParameters` has 1 entry { "AuthScheme": "token68" } which doesn't seem correct? 

I'd would think you either add this param in `RawParameters` as { "token68": null } or {"token68" : "token68"} 

@pmaytak, since you also reviewed this - thoughts?

Reply:
Updated logic here.

Reply:
Is token68 an actual type of a token...? If so `{"token68" : "ActualToken68Token"}` makes sense. I'll take another look.